Fixes bug where <font> element is never closed

juergenmueller77 opened the pull request:
Files changed in commit:
SupplierAllocations.php
On branch develop

Remove <font> element and replace with class for the <div>
element meaning each theme displays this differently

juergenmueller77 committed:
Fixes bug where <font> element is never closed

Files changed in commit:
SupplierAllocations.php
On branch develop

Remove <font> element and replace with class for the <div>
element meaning each theme displays this differently

timschofield commented:
Hey Jüergen Ich kann das Problem verstehen, dass du festlegst. Ich denke, dass das Problem mit page_help_text darin besteht, dass der Benutzer die Seitenhilfe ausblenden kann. Vielleicht brauchen wir eine neue Klasse für Text, der immer angezeigt werden muss, vielleicht page_hint_text? - Tim

juergenmueller77 commented:
Ja, ok, lass mich es versuchen

juergenmueller77 committed:
Create a new css class for persistant help text

Files changed in commit:
SupplierAllocations.php
css/aguapop/default.css
css/arc-dark/default.css
css/default/default.css
css/fluid/default.css
css/fresh/default.css
css/gel/default.css
css/mobile/default.css
css/professional-rtl/default.css
css/professional/default.css
css/silverwolf/default.css
css/wood/default.css
css/xenos/default.css
On branch develop

Create css class page_hint_text in all themes
Use in SupplierAllocations.php to show the hint

juergenmueller77 commented:
Es ist getan wie sie es gewünscht haben

timschofield commented:
Danke, ich werde es später überprüfen
